tutorial/branches/project-diff.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Project diff
+permalink: /tutorial/branches/project-diff/
+---
+
+In the previous parts you tested how [files](/tutorial/branches/files-in-branch/) and [tables](/tutorial/branches/tables-in-branch/) behave in branches. Now it's time to complete the branch lifecycle and merge the development branch back to production. 
+
+### Show Project Diff
+First, you want to check what are the things that you changed against the production. Go to the dashboard and click the button **Show Project Diff** on the right. There you can see the things you’ve changed.
```

Comment:
tady je trochu nepříjemná situace, že pokud jsem udělal files i tables, tak mám dvě branche 

Tzn. tady bych dělal diff tabulkové branche s poznámkou, že diff file branche vypadá analogicky.

Reply:
Tyjo, to mi vlastně vůbec nedošlo. Hlavní důvod je ten, že jsem produkci měl vždycky už hotovou a na čisto jsem si říkal, že to udělám jen jednou. Druhé možné řešení (možná lepší) by bylo udělat nejdřív oboje přípravné kroky a pak udělat ty změny do files i tables v branchích.

Reply:
Tohle budu muset nějak upravit.

Reply:
Myslím, že obě řešení jsou přijatelný, ale je potřeba se k tomu otevřeně přiznat a netvářit se, že je tam jen jedna branch